plugin/webhook/webhook.go

```diff
@@ -86,6 +86,18 @@ func (c *Context) getMetaList() []meta {
 			Name:  "Issue",
 			Value: c.Issue.Name,
 		})
+		// For VCS workflow, the generated issue description is composed of file names in the push event.
+		// So the description could be long, which is hard to display if merged into the issue name.
+		// We also trim it to 200 bytes to limit the message size in the webhook body, so that users can
+		// view it easily in the corresponding webhook client.
+		description := c.Issue.Description
+		if len(description) > 200 {
+			description = description[:200] + "... (view details in Bytebase)"
```

Comment:
This assumed descriptions are written in ASCII, which doesn't work for Chinese characters for example.